services/hh/service.go

```diff
@@ -165,8 +172,19 @@ func (s *Service) expireWrites() {
 	}
 }
 
-// purgeWrites will cause the handoff queues to remove writes that are no longer
-// valid.  e.g. queued writes for a node that has been removed
-func (s *Service) purgeWrites() {
-	panic("not implemented")
+// deleteQueues will cause the service to remove queues for inactive nodes.
+func (s *Service) deleteQueues() {
+	defer s.wg.Done()
+	ticker := time.NewTicker(time.Hour)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-s.closing:
+			return
+		case <-ticker.C:
+			if err := s.HintedHandoff.DeleteOlderThan(time.Duration(s.cfg.MaxAge)); err != nil {
```

Comment:
`PurgeOlderThan` and `DeleteOlderThan` are very similarly named and might cause confusion in the future.  Maybe `PurgeInactiveOlderThan`?  Dunno...